apps/basic-example/src/NativeDetector.tsx

```diff
@@ -18,9 +18,7 @@ export default function App() {
   );
 
   const gesture = useGesture('PanGestureHandler', {
-    onGestureHandlerAnimatedEvent: event,
-    onGestureHandlerEvent: (e: any) =>
-      console.log('onGestureHandlerEvent', e.nativeEvent),
+    onUpdate: event,
```

Comment:
Not the best, but still better than example crashing on passing `onGestureHandlerAnimatedEvent`